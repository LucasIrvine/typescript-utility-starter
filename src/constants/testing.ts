// THIS FILE IS EXCLUDED FROM THE LIB BUILD AND IS USED ONLY FOR TESTING
export const EXAMPLE_TEST_STRING = 'test';
