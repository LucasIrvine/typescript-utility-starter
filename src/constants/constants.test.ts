import { EMPTY_STRING } from './constants';

describe('constants', () => {
  describe('.EMPTY_STRING', () => {
    it('should test computed constants so youo know if a child constant changes anything', () => {
      expect(EMPTY_STRING).toMatchSnapshot();
    });
  });
});
