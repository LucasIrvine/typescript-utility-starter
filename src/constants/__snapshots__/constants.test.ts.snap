// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants .EMPTY_STRING should test computed constants so youo know if a child constant changes anything 1`] = `""`;
